operations/workplace/people/README.md

```diff
@@ -9,9 +9,9 @@ Description: >-
 
 | Name | Title |
 | :--- | :--- |
-| [Natalie Jew](https://community.mattermost.com/core/messages/natalie.jew) |VP, Human Resources|
+| [Natalie Jew](https://community.mattermost.com/core/messages/natalie.jew) |VP, People|
```

Comment:
```suggestion
| [Natalie Jew](https://community.mattermost.com/core/messages/natalie.jew) |VP, Human Resources|
```

Reply:
Thanks @lynn915 , Natalie confirmed her title hasn't changed and I've reverted to the existing title.